lib/internals/dynamics.js

```diff
@@ -133,68 +134,66 @@ module.exports = (
   const parseAuthzRoles = (authzApiResponse) => {
```

Comment:
Does this always expect roles for a particular user? If so could that be included in the JSDoc?

Reply:
It expects the response from the authz api - which is in the context of a user and service

Reply:
OK as someone who hasn't written any code to do with the AuthZ stuff thats something I'd have to go to another part of the codebase and check